src/backend/main.ts

```diff
@@ -79,3 +83,23 @@ function hash(input: string, pepper: string) {
     .update(input + pepper)
     .digest('base64');
 }
+
+export async function storeTotalResponsesToS3() {
```

Comment:
Wrt. `async/await`, I'm fine with either that or naked Promises, but I don't think we should mix the two styles.

Reply:
That said, my desire for this to end up on `master` might be too great... :wink:

Reply:
I have no objections to change this to `Promise.resolve` flow, though personally I've started to favor `async` functions quite highly for a few reasons:

- They clearly indicate that the function is doing some async stuff. I even use the keyword when I'm simply returning a promise, i.e. not even using `await` in the function. I'll refer you to a nice article _What color if your function_ https://journal.stuffwithstuff.com/2015/02/01/what-color-is-your-function/
- Reduces boilerplate, though not much. One handy thing is that it solves the ergonomics for block functions that return a value as `then` handlers, by just getting rid of that paradigm.
- It's already standard in JS, you can use `async` functions in devtools console and Node.js. I also heard rumours that some browsers accept `await` as a top-level statement, very useful in console. Use the platform, Luke.

The only thing that really remains a a small problem in general is wrapping your head around the fact that the code breaks into two whenever you call `await`. The code above it (well up to the previous `await` at least) executes synchronously and the code following `await` is executed at some later point in time. Though this parallels how for e.g. generators work in JS, so it's something that one should just get it's mind set right.

Reply:
As a follow-up, I'm willing to convert those other backend functions to `async`/`await` as well ;)

Reply:
OK, SOLD to the man willing to put his ~money~ patches where his mouth is! :smile: 

Let's merge this in, and do the conversion in a follow-up PR. :+1: